recipe/meta.yaml

```diff
@@ -40,7 +40,8 @@ requirements:
   run:
     - python
     - numpy
-    - scipy >=1.8
+    - scipy >=1.8,<1.9.2  # [py <= 39]
+    - scipy >=1.8  # [py > 39]
```

Comment:
See https://github.com/scikit-image/scikit-image/blob/v0.20.0/pyproject.toml#L40-L41

Reply:
That constraint was probably a mistake; we're getting rid of it in the next version, being released this week.

Reply:
ah good spotting that! I have been scratching me head for the past hour why our Python 3.8 and 3.9 tests die with a fairly obtuse`ContextualVersionConflict` (see [fails](https://github.com/ESMValGroup/ESMValTool/actions/runs/4677958708/jobs/8286095951) ) :grin: